### PR TITLE
PCI-DSS profile for fedora

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Add PCI-DSS profile to Fedora
- The Profile is almost in parity with RHEL7
  - The only difference is check and fix for`smartcard_auth` rule, which is missing in Fedora.

### CODE REVIEW

**Comment by yuumasato** [comment]
This is close to completion, there still some rules that I need to check the remediation, but the results look good. 
Here are scan reports of Fedora 28:
* before remediation: [pci-dss-fedora.html.txt](https://github.com/OpenSCAP/scap-security-guide/files/2324643/pci-dss-fedora.html.txt)
* after remediation: [remediated-pci-dss-fedora.html.txt](https://github.com/OpenSCAP/scap-security-guide/files/2324644/remediated-pci-dss-fedora.html.txt)

File: linux_os/guide/system/software/updating/ensure_fedora_gpgkey_installed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+severity: high
+
+identifiers:
+    cce@rhel6: 26506-6
+    cce@rhel7: 26957-1
+
+references:
+    stigid@rhel6: RHEL-06-000008
+    srg@rhel6: SRG-OS-000090
+    disa@rhel6: '351'
```
Comment (redhatrises) [inline linux_os/guide/system/software/updating/ensure_fedora_gpgkey_installed/rule.yml]: Lines 25-26 and 29-31 shouldn't exist.

File: linux_os/guide/system/software/updating/ensure_fedora_gpgkey_installed/rule.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    cis: 1.2.3
+    cjis: 5.10.4.1
+    cui: 3.4.8
+    disa: "1749"
+    hipaa: 164.308(a)(1)(ii)(D),164.312(b),164.312(c)(1),164.312(c)(2),164.312(e)(2)(i)
+    nist: CM-5(3),SI-7,MA-1(b)
+    ospp@rhel7: FAU_GEN.1.1.c
+    pcidss: Req-6.2
+    srg: "366"
```
Comment (redhatrises) [inline linux_os/guide/system/software/updating/ensure_fedora_gpgkey_installed/rule.yml]: This srg doesn't look right to me.

Comment (yuumasato) [reply linux_os/guide/system/software/updating/ensure_fedora_gpgkey_installed/rule.yml]: @redhatrises I don't know what to put there, nor where to get the right value, so I'll be removing the `srg` field.

**Comment by redhatrises** [comment]
@yuumasato is this no longer a work in progress?

**Comment by yuumasato** [comment]
@redhatrises There are some issues with remediation that I would like to iron out, so I guess this is stil WIP, sorry.

In the mean time, check scan results of a Fedora 28 VM after remediation:
[xccdf_org.ssgproject.content_profile_pci-dss-remediation.html.renamed.txt](https://github.com/OpenSCAP/scap-security-guide/files/2331910/xccdf_org.ssgproject.content_profile_pci-dss-remediation.html.renamed.txt)

**Comment by yuumasato** [comment]
@redhatrises actually, I think we can work on fixing remediations later.

**Comment by redhatrises** [comment]
@yuumasato great job! Ack.

**Pull request merged**